lib/class-vip-request-block.php

```diff
@@ -25,15 +25,24 @@ class VIP_Request_Block {
 	 * @param string $value target IP address to be blocked.
 	 */
 	public static function ip( string $value ) {
+		// Don't run validation if the passed value is not a valid IP
+		if ( ! filter_var( $value, FILTER_VALIDATE_IP ) ) {
+			return;
+		}
+
```

Comment:
Adding this because now we're matching substrings with `stripos`. If the user would pass `1` as `$value` and the IP is `1.1.1.1`, we would block the request even though `1` is not a valid IP address.